app/core/services/stt_google_v2.py

```
@@ -261,4 +330,49 @@ def req_gen():
+        if not audio_bytes:
+            return
+
+        request = cloud_speech.RecognizeRequest(
+            recognizer=self._recognizer,
+            config=recog_cfg,
+            content=audio_bytes,
+        )
+
+        try:
+            resp = await asyncio.to_thread(self.client.recognize, request=request)
+        except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)